src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/RestoreAzureRMRecoveryServicesBackupItem.cs

```diff
@@ -338,8 +338,7 @@ public override void ExecuteCmdlet()
                     }
                     else
                     {
-                        ARSVault vault = ServiceClientAdapter.GetVault(resourceGroupName, vaultName);
-                        secondaryRegion = BackupUtils.regionMap[vault.Location];
+                        throw new PSArgumentException(Resources.VaultLocationRequiredForCRR);
```

Comment:
This piece of code looks like never reached.

Reply:
this code may reach when RestoreToSecondaryRegion switch is given but VaultLocation isn't given.